src/hevc.h

```diff
@@ -0,0 +1,31 @@
+/*
+    FLVMeta - FLV Metadata Editor
+
+    Copyright (C) 2007-2019 Marc Noirot <marc.noirot AT gmail.com>
+
+    This file is part of FLVMeta.
+
+    FLVMeta is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    FLVMeta is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with FLVMeta; if not, write to the Free Software
+    Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+*/
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "flv.h"
+#include "types.h"
+
+#define HEVC_MAX_SUB_LAYERS 7
+
+int read_hevc_resolution(flv_video_tag * vt, flv_stream * f, uint32 body_length, uint32 * width, uint32 * height);
```

Comment:
Could you please wrap these in an `extern C` conditional block like in other header files?